### PR TITLE
Update register-devices-partner.md

### PR DESCRIPTION
**Pull request by msbemba**
Both the scripts are similar and it works for partners as well - So removed the part "he Windows PowerShell script is different from the one used for the Microsoft Managed Desktop Admin portal." from not. 

Also added a note about editing the file in excel and saving as a CSV will not generate a usable file -Extra columns are not supported. Quotes are not supported. Only ANSI-format text files can be used (not Unicode). Headers are case-sensitive. Editing the file in Excel and saving as a CSV file will not generate a usable file due to these requirements.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5506

File: microsoft-365/managed-desktop/get-started/register-devices-partner.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Once you have established the relationship with your customers, you can use Part
...
+> You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot](/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). Extra columns are not supported. Quotes are not supported. Only ANSI-format text files can be used (not Unicode). Headers are case-sensitive. Editing the file in Excel and saving as a CSV file will not generate a usable file due to these requirements. Please make sure that any leading zeroes are preserved in the device serial numbers. Partners should use [Get-WindowsAutoPilotInfo](https://www.powershellgallery.com/packages/Get-WindowsAutoPilotInfo) to register devices for Microsoft Managed Desktop devices in Partner Center.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/register-devices-partner.md]:
```suggestion
> You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot](/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). Extra columns are not supported. Quotes are not supported. Only ANSI-format text files can be used (not Unicode). Headers are case-sensitive. Editing the file in Excel and saving it as a CSV file will not generate a usable file due to these requirements. Please make sure that any leading zeroes are preserved in the device serial numbers. Partners should use [Get-WindowsAutoPilotInfo](https://www.powershellgallery.com/packages/Get-WindowsAutoPilotInfo) to register devices for Microsoft Managed Desktop devices in Partner Center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

**Comment by jaimeo** [comment]
@msbemba @JohanFreelancer9 : we do not use "please."

**Comment by jaimeo** [comment]
@jwmurraycmu are you OK with these edits w/r/t technical accuracy?

**Comment by jaimeo** [comment]
@AlexBlain are you OK with these edits w/r/t technical accuracy?

**Comment by AlexBlain** [comment]
@jaimeo fine with that

**Pull request merged**